app/javascript/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <div className="App">
+      <Greeting />
+    </div>
+  );
```

Comment:
- [ ] Let's set a route to the Greeting component with react-router-dom as required in the project requirement. Please add a route that route to the Greeting compone not from the App. 👍🏽

<img width="814" alt="Screenshot 2023-08-16 at 9 57 48 AM" src="https://github.com/Kwaku28/hello_rails_react/assets/50941074/ea1f726c-207a-48cb-9b77-59a24e93fc2f">